Description: Patch to include layout option for python in Debian
Author: Mario Limonciello <superm1@ubuntu.com>
Forwarded: not-needed

==========================================================================
Index: mythtv-master/mythtv/bindings/python/Makefile
===================================================================
--- mythtv-master.orig/mythtv/bindings/python/Makefile	2011-03-06 22:31:46.000000000 -0600
+++ mythtv-master/mythtv/bindings/python/Makefile	2011-04-09 22:57:12.162532463 -0500
@@ -1,7 +1,7 @@
 include ../../config.mak
 
 ifdef INSTALL_ROOT
-	ROOT_FLAGS = --root="$(INSTALL_ROOT)"
+	ROOT_FLAGS = --root="$(INSTALL_ROOT)/../libmyth-python"
 else
 ifndef USE_PYTHON_PIP
 	ROOT_FLAGS = --root="/"
@@ -14,6 +14,12 @@ ifneq ($(PREFIX:/=), /usr)
 	endif
 endif
 
+ifdef USE_PYTHON_PIP
+	PREFIX_FLAGS=DEB_PYTHON_INSTALL_LAYOUT=deb_system
+else
+	PREFIX_FLAGS="--install-layout=deb"
+endif
+
 PIP_OPTIONS = --no-build-isolation --no-cache-dir --no-index --disable-pip-version-check --no-deps
 WHEEL_DIR = dist
 
@@ -32,7 +38,7 @@ python_build:
 	$(PYTHON) -m pip wheel $(PIP_OPTIONS) --wheel-dir ./$(WHEEL_DIR) .
 
 install:
-	$(PYTHON) -m pip install $(ROOT_FLAGS) $(PREFIX_FLAGS) $(PIP_OPTIONS) --find-links ./$(WHEEL_DIR) MythTV
+	$(PREFIX_FLAGS) $(PYTHON) -m pip install $(ROOT_FLAGS) $(PIP_OPTIONS) --root-user-action ignore --find-links ./$(WHEEL_DIR) MythTV
 
 uninstall:
 	$(warning python pip uninstall is not supported for python bindings)
Index: mythtv-master/mythtv/bindings/perl/Makefile
===================================================================
--- mythtv-master.orig/mythtv/bindings/perl/Makefile	2011-03-06 22:31:46.000000000 -0600
+++ mythtv-master/mythtv/bindings/perl/Makefile	2011-04-09 22:57:12.162532463 -0500
@@ -1,7 +1,7 @@
 include ../../config.mak
 
 ifdef INSTALL_ROOT
-ROOT_FLAGS = PERL_INSTALL_ROOT="$(INSTALL_ROOT)"
+ROOT_FLAGS = PERL_INSTALL_ROOT="$(INSTALL_ROOT)/../libmythtv-perl"
 endif
 
 ifneq ($(PREFIX:/=), /usr)
Index: mythtv-master/mythtv/bindings/php/Makefile
===================================================================
--- mythtv-master.orig/mythtv/bindings/php/Makefile	2011-04-09 22:56:43.000000000 -0500
+++ mythtv-master/mythtv/bindings/php/Makefile	2011-04-09 22:57:49.806719139 -0500
@@ -4,7 +4,7 @@
 FILES += MythTV.php MythTVProgram.php MythTVRecording.php
 FILES += MythTVStorageGroup.php
 
-INSTDIR = ${INSTALL_ROOT}${PREFIX}/share/mythtv/bindings/php
+INSTDIR = ${INSTALL_ROOT}/../php-mythtv${PREFIX}/share/mythtv/bindings/php
 
 all:
 clean:
